vault/dendron.tutorial.taking-notes.md

```diff
@@ -1,54 +1,97 @@
 ---
 id: 4RFQMNA91cRW12VJl_PFd
 title: Taking Notes
-desc: Taking Notes
-updated: 1636527192313
+desc: Creating notes, understanding hierarchy, and using Lookup to quickly find your notes
+updated: 1639427284878
 created: 1625563944736
 nav_order: 1
 ---
 
 ### Create a Note
 
-To create a note, use `Ctrl+L` / `Cmd+L` to bring up Dendron's lookup. This is a shortcut for the `Dendron: Lookup` command.
+To create a note, use `Ctrl+L` / `Cmd+L` to bring up [[Dendron's Lookup|dendron://dendron.dendron-site/dendron.topic.lookup]]. This is a shortcut for the `Dendron: Lookup` command.
 
-> 💡 The lookup command is the main way to interact with Dendron. It is used for both looking up your notes and creating new notes. When you do a lookup on a note that hasn't been created, Dendron will create it for you. Remember the `Ctrl+L` / `Cmd+L` shortcut.
+> 💡 **TIP:** The lookup command is the main way to interact with Dendron. It's used for looking up and creating notes. When you do a lookup on a note that hasn't been created, Dendron will create it for you.
 
-> 🌱 Try it yourself - bring up the lookup bar with `Ctrl+L` / `Cmd+L`. Type `recipes` and hit Enter. Afterward, you can use the `Ctrl+Tab` / `Cmd+Tab` shortcut to get back to the previous note.
+> 🌱 **ACTION:** Try it yourself! Open the lookup bar with `Ctrl+L` / `Cmd+L`. Type `recipes` and hit `Enter`. Afterward, you can use the `Ctrl+Tab` / `Cmd+Tab` shortcut to get back to the previous note you were viewing.
 
-This should have created a note named `recipes.md`. Notes in Dendron are just plaintext markdown. They live in your file system and are portable across any platform, which makes it very easy to import/ export your notes and even to publish your notes to a hosted website. You can try to add some content to the body of your `recipes` note now.
+This creates a note named `recipes.md`. Notes in Dendron are just plaintext [[markdown|dendron://dendron.dendron-site/dendron.topic.markdown]]. Notes:
 
-The --- section at the top of each note is frontmatter. Frontmatter are custom attributes at the top of each markdown file. Dendron uses it to store metadata about each note for features like publishing. Don't modify the id attribute on the front matter.
+- Live locally on your file system.
+- Are portable across operating systems and a variety of software.
+- Make importing/exporting easier, including [[publishing your notes to a hosted website|dendron://dendron.dendron-site/dendron.topic.publish]].
+
+You can try to add some content to the body of your `recipes` note now.
+
+#### Frontmatter
+
+The `---` section at the top of each note is **[[frontmatter|dendron://dendron.dendron-site/dendron.topic.frontmatter]]**. Frontmatter contains custom attributes at the top of each markdown file. Dendron uses it to store metadata about each note for features like publishing.
```

Comment:
might be worthwile to add as `## Summary` to dendron.topic.frontmatter.md

You'll want to update the intro to frontmatter if you do